src/main/kotlin/racing/model/Car.kt

```diff
@@ -1,15 +1,20 @@
 package racing.model
 
-data class Car constructor(private val position: Int) {
+data class Car(
+    private val position: Int,
+    private val name: String = "unknown"
+) : Comparable<Car> {
 
     init {
         require(0 <= this.position) { INVALID_POSITION_EXCEPTION }
     }
 
-    override fun toString(): String = PRINT_STEP_SIGN.repeat(this.position)
+    override fun compareTo(other: Car): Int = position - other.position
+
+    override fun toString(): String = "$name: ${PRINT_STEP_SIGN.repeat(this.position)}"
 
     fun moveForward(randomValue: Int): Car {
-        return spawnAt(position + this.getStep(randomValue))
+        return Car(position + this.getStep(randomValue), name)
```

Comment:
```suggestion
        return Car(position + getStep(randomValue), name)
```
this는 생략해주셔도 될 것 같아요.